dicom_numpy/combine_slices.py

```diff
@@ -77,13 +77,24 @@ def combine_slices(datasets, rescale=None):
     return voxels, transform
 
 
+def sort_by_slice_position(slice_datasets):
```

Comment:
Nice docstring!

In order to make this show up in the "Read the docs" page, I think we will also need to add something here https://github.com/innolitics/dicom-numpy/blob/master/docs/source/index.rst#details

Reply:
Just added an `autofunction` line for this.